lib/pdfkit/html_preprocessor.rb

```diff
@@ -0,0 +1,23 @@
+class PDFKit
+  class HtmlPreprocessor
```

Comment:
Why is this a class and not a module. There's nothing being stored here at all and there's no state.